lib/utils.js

```diff
@@ -81,9 +128,8 @@ function mergeQuery(base, update, spec) {
     if (!base.include) {
       base.include = update.include;
     } else {
-      var saved = base.include;
-      base.include = {};
-      base.include[update.include] = saved;
+      //merge inclusion
+      base.include = mergeIncludes(base.include, update.include);
```

Comment:
Thanks @raymondfeng  for pointing out. I'll push it along with the test fix.
However, it seems mergeQuery is used for inclusion nesting.
```js
physician.patients.getAsync({include: 'address'})
```
here - scope applied query should be 
```js
{
  collect:'patient',
  include:{
    patient:'address'
  }
}
```
but because of the fix, it is
```js
{
  collect:'patient',
  include:{
    patient:true,
    address:true
  }
}
```
But it works if I don't use getAsync as shown below.
```js
physician.patients({include: 'address'})
```

Any pointers?